src/sorcha/sorcha.py

```diff
@@ -89,11 +94,28 @@ def runLSSTPostProcessing(cmd_args):
     verboselog("Random number seed is {}.".format(rng_seed))
     rng = np.random.default_rng(rng_seed)
 
+    # Set up the data readers.
+    reader = CombinedDataReader(verbose=True)
+
+    if cmd_args["makeTemporaryEphemerisDatabase"] or cmd_args["readTemporaryEphemerisDatabase"]:
+        reader.add_ephem_reader(DatabaseReader(cmd_args["readTemporaryEphemerisDatabase"]))
+    else:
+        # TODO: Once more ephemerides_types are added this should be wrapped in a EphemerisDataReader
+        # That does the selection and checks. We are holding off adding this level of indirection until there
+        # is a second ephemerides_type.
+        ephem_type = configs["ephemerides_type"]
+        if ephem_type.casefold() != "oif":  # pragma: no cover
+            pplogger.error(f"PPReadAllInput: Unsupported value for ephemerides_type {ephem_type}")
+            sys.exit(f"PPReadAllInput: Unsupported value for ephemerides_type {ephem_type}")
+        reader.add_ephem_reader(OIFDataReader(cmd_args["oifoutput"], configs["eph_format"]))
+
+    reader.add_aux_data_reader(OrbitAuxReader(cmd_args["orbinfile"], configs["aux_format"]))
+    reader.add_aux_data_reader(CSVDataReader(cmd_args["paramsinput"], configs["aux_format"]))
+    if configs["comet_activity"] == "comet":
+        reader.add_aux_data_reader(CSVDataReader(cmd_args["cometinput"], configs["aux_format"]))
```

Comment:
This looks so good. One thing that has come up with the light curve variability work is that there's a reasonably good chance that users will create new functions that require parameters beyond what is currently read in. 

This works sets us up so nicely for that.